src/dev/flang/parser/Parser.java

```diff
@@ -1436,32 +1436,19 @@ Expr call(Expr target)
 call        : name actuals callTail
             ;
 actuals     : actualArgs
-            | dot NUM_LITERAL
+            | dot select callTail
```

Comment:
callTail is already after `actuals`, so this rule (`dot select callTail`) should be removed completely.